tests/transformers/test_transformers_model_export.py

```diff
@@ -52,6 +58,22 @@
 _IMAGE_PROCESSOR_API_CHANGE_VERSION = "4.26.0"
 
 
+def clean_cache(threshold=2):
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
def clean_cache(threshold=2):
```

Can we use `autouse`?

Reply:
yep! Updated and removed all previous func calls